front/app/api/custom_form/useCustomForm.ts

```diff
@@ -8,8 +8,8 @@ import { CustomFormKeys, ICustomForm, ICustomFormParameters } from './types';
 
 const fetchCustomForm = ({ projectId, phaseId }: ICustomFormParameters) => {
   const apiEndpoint = phaseId
```

Comment:
Apparently, the project variant is in place because when we update the form in an ideation project, we want to update the form across all phases. Wondering if it would be easier to only have custom_form tied to a phase and then update it for all phases. Wouldn't that make our code easier?